UnitsNet.Serialization.JsonNet/UnitsNetJsonConverter.cs

```diff
@@ -93,7 +93,10 @@ public class UnitsNetJsonConverter : JsonConverter
             object unit = Enum.Parse(reflectedUnitEnumType, unitEnumValue);
 
             // Mass.From() method, assume no overloads exist
-            MethodInfo fromMethod = reflectedUnitType.GetMethod("From");
+            MethodInfo fromMethod = (from m in reflectedUnitType.GetMethods()
+                                     where m.Name.Equals("From", StringComparison.InvariantCulture) &&
+                                     !m.ReturnType.IsGenericType  // we want the non nullable type
```

Comment:
The tests captured this, hence the change (I would never have discovered it without the tests). Since there is a test that checks that this work I would leave it as is. We could add `m.ReturnType.IsValueType && Nullable.GetUnderlyingType(m.ReturnType) == null` but what's there right now works and it seems unlikely that more functions named `From` are added.